themes/material/default/selectidp-links.php

```diff
@@ -100,11 +104,15 @@ function clickedAnyway(idpName) {
             ?>
             <div class="mdl-card mdl-shadow--2dp disabled row-aware" title="<?= $hoverText ?>"
                  onclick="clickedAnyway('<?= $name ?>')">
-                <div class="mdl-card__media white-bg fixed-height" layout-children="row"
-                     child-spacing="center">
-                    <img class="scale-to-parent" id="<?= $idpId ?>"
-                         src="<?= empty($idp['logoURL']) ? 'default-logo.png'
-                                                         : $idp['logoURL'] ?>">
+                <div class="mdl-card__media white-bg fixed-height" layout-children="row" child-spacing="center">
+                    <div class="logo-container fill-parent">
+                        <div class="image-wrapper">
+                            <img class="logo" id="<?= $idpId ?>"
+                                src="<?= empty($idp['logoURL']) ? 'default-logo.png'
+                                                                : $idp['logoURL'] ?>">
+                        </div>
+                        <span class="mdl-color-text--grey-600 logo-caption" ><?= empty($idp['logoCaption']) ? '<br>': $idp['logoCaption'] ?> </span>
```

Comment:
For the sake of proper-ness, it's probably worth HTML-encoding the logo caption, to ensure it displays properly (even if it at some point contains meaningful HTML symbols, like an `&`):
```suggestion
                        <span class="mdl-color-text--grey-600 logo-caption" ><?= empty($idp['logoCaption']) ? '<br>': htmlentities($idp['logoCaption']) ?> </span>
```